libs/core-ui/src/lib/Interfaces/IDataBalanceMeasures.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export interface IDataBalanceMeasures {
+  AggregateBalanceMeasures?: IAggregateBalanceMeasures;
+  DistributionBalanceMeasures?: IDistributionBalanceMeasures;
+  FeatureBalanceMeasures?: IFeatureBalanceMeasures;
+}
+
+export interface IAggregateBalanceMeasures {
+  AtkinsonIndex: number;
+  TheilLIndex: number;
+  TheilTIndex: number;
+}
+
+export interface IDistributionBalanceMeasures {
+  [key: string]: IDistributionBalanceMeasure;
+}
+
+export interface IDistributionBalanceMeasure {
+  ChiSquarePValue: number;
+  ChiSquareStat: number;
+  InfiniteNormDist: number;
+  JensenShannonDist: number;
+  KLDivergence: number;
+  TotalVarianceDist: number;
+  WassersteinDist: number;
+  CrossEntropy: number;
+}
+
+export interface IFeatureBalanceMeasures {
+  [key: string]: IFeatureBalanceMeasure[];
+}
+
+export interface IFeatureBalanceMeasure {
+  ClassA: string;
+  ClassB: string;
+  StatisticalParity: number;
```

Comment:
That one is just for me, but I would love to see the definition 🙂

Reply:
The definition will be in part 2, but [here is a sneak peek](https://github.com/ms-kashyap/responsible-ai-widgets/pull/5/files#diff-a95892d97ed2f98a5605dbf0300e2a67fd753a8653b71a49336084db08f74fff) to the definition from the whole feature PR in my own fork 😊

Please feel free to put comments there if you'd like!